### PR TITLE
Add instruction for installing fapolicyd on an existing install

### PR DESCRIPTION
**Pull request by ehelms**
* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13
* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Comment by asteflova** [comment]
For the sake of completeness, the bug report that spawned this PR is https://issues.redhat.com/browse/SAT-26834  (It can be useful when reviewing the update.)

File: guides/common/modules/proc_installing-fapolicyd-on-server.adoc [hunk 1]
```diff
@@ -19,12 +19,18 @@ endif::[]
...
 
 .Procedure
-. Install fapolicyd:
+. For a new installation, install fapolicyd:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # dnf install fapolicyd
 ----
+. For an existing installation, install fapolicyd using {foreman-maintain}:
```
Comment (Lennonka) [inline guides/common/modules/proc_installing-fapolicyd-on-server.adoc:28]:
```suggestion
. For an existing installation, install fapolicyd by using {foreman-maintain}:
```

**Review by Lennonka — CHANGES_REQUESTED** [review]
Perhaps we should also cherry-pick it to all versions 3.9+?

Otherwise looks good.

File: guides/common/modules/proc_installing-fapolicyd-on-server.adoc [hunk 2]
```diff
@@ -19,12 +19,18 @@ endif::[]
...
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages install fapolicyd
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-fapolicyd-on-server.adoc:32]:
```suggestion
# {project-package-install} fapolicyd
```

upstream+orcharhino: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L187
Satellite: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L95

reason: only Satellite uses "foreman-maintain" to wrap package installation. Please also adjust the introductory sentence.

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

**Review by maximiliankolb — COMMENTED** [review]
I am not sure about this yet. I suggest the following diff to "master":

````diff
diff --git a/guides/common/assembly_installing-server-connected.adoc b/guides/common/assembly_installing-server-connected.adoc
index 8b1da7a6dd..7ca62fe4d8 100644
--- a/guides/common/assembly_installing-server-connected.adoc
+++ b/guides/common/assembly_installing-server-connected.adoc
@@ -14,6 +14,8 @@ endif::[]
 
 include::modules/proc_configuring-repositories.adoc[leveloffset=+1]
 
+include::modules/proc_installing-the-satellite-server-packages.adoc[leveloffset=+1]
+
 // we do not package "fapolicy" for Foreman on Debian/Ubuntu
 ifndef::foreman-deb[]
 include::modules/con_using-fapolicyd-on-server.adoc[leveloffset=+1]
@@ -21,8 +23,6 @@ include::modules/con_using-fapolicyd-on-server.adoc[leveloffset=+1]
 include::modules/proc_installing-fapolicyd-on-server.adoc[leveloffset=+2]
 endif::[]
 
-include::modules/proc_installing-the-satellite-server-packages.adoc[leveloffset=+1]
-
 ifdef::foreman-el,katello,satellite[]
 include::modules/proc_synchronizing-the-system-clock-with-chronyd.adoc[leveloffset=+1]
 endif::[]
diff --git a/guides/common/modules/proc_installing-fapolicyd-on-server.adoc b/guides/common/modules/proc_installing-fapolicyd-on-server.adoc
index 2e53ac28ef..e961bce00f 100644
--- a/guides/common/modules/proc_installing-fapolicyd-on-server.adoc
+++ b/guides/common/modules/proc_installing-fapolicyd-on-server.adoc
@@ -23,7 +23,7 @@ endif::[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# dnf install fapolicyd
+# {project-package-install} fapolicyd
 ----
 . Start the `fapolicyd` service:
 +
````

This ensures that on Satellite, users will always use "satellite-maintain" to install packages. Does this achive your goal?

However, looking at https://docs.theforeman.org/nightly/Installing_Server/index-satellite.html#synchronizing-the-system-clock-with-chronyd_satellite tells me that you install chronyd with "dnf" and "sos" with `satellite-maintain`. Is this intentional?

Is there any technical reason to install fapolicyd _before_ installing Foreman/Katello? If not, I suggest to move the procedure as shown in the diff above.

**Comment by ehelms** [comment]
> Is there any technical reason to install fapolicyd _before_ installing Foreman/Katello? If not, I suggest to move the procedure as shown in the diff above.

Yes. On new installations, `fapolicyd` should be installed before.

**Review by maximiliankolb — APPROVED** [review]
Thanks for clarifying; then I'd say this is OK as is. Handing over to Lena.

**Review by Lennonka — APPROVED** [review]
LGTM, but I'm still fuzzy about the cherrypicking.

**Comment by ehelms** [comment]
> LGTM, but I'm still fuzzy about the cherrypicking.

No cherry picking is required.

**Comment by maximiliankolb** [comment]
> > LGTM, but I'm still fuzzy about the cherrypicking.
> 
> No cherry picking is required.

The PR is for "master", so Foreman nightly. You've selected "Foreman 3.9" in the description. Can we merge to "master" and cherry-pick to "3.11", "3.10", and "3.9"?

**Comment by ehelms** [comment]
> The PR is for "master", so Foreman nightly. You've selected "Foreman 3.9" in the description. Can we merge to "master" and cherry-pick to "3.11", "3.10", and "3.9"?

Oh maybe I did do that, apologies. Feel free to merge and cherry pick to those.

**Comment by Lennonka** [comment]
Cherry-picked:
- a849a427f4..33cb5b4b05  3.11 -> 3.11
- 1577af2b3d..17c38d21b9  3.10 -> 3.10
- 85a6135468..8b78c512ff  3.9 -> 3.9

**Pull request merged**